tests/test_username_api.py

```diff
@@ -1,24 +1,29 @@
-import unittest
 import json
-import yaml
-import string
+import logging
 import random
+import string
+import os.path
+
+import yaml
 import pytest
-import logging
+from parameterized import parameterized
+
 import username_api
-import os.path
 
 data = yaml.load(open(os.path.join('tests', 'test_data.yml')))
 websites = yaml.load(open('websites.yml'))
 
 invalid_username = '$very%long{invalid}user(name)'
 
-def generate_random_username(length, first_char_letter=False):
-  return ''.join(generate_random_character(length, first_char_letter))
+def generate_random_username(website, length):
+  if website == 'openhub':
+    first_char_letter = True
+  return ''.join(generate_random_character(length,
+    first_char_letter=first_char_letter))
 
-def generate_random_character(length, first_char_letter):
+def generate_random_character(length, **kwargs):
```

Comment:
there is no benefit in using kwargs here.

and fiddling with this random generator is not related to your issue.

Reply:
I used kwargs since we can add more option later. It's quite possible.

Reply:
No, thank you.  This is unrelated to this change.
This isnt even the right way to do it.
I've created https://github.com/manu-chroma/username-availability-checker/issues/47 for that.

Reply:
got it.